balancer.go

```diff
@@ -395,3 +395,40 @@ func (rr *roundRobin) Close() error {
 	}
 	return nil
 }
+
+type pickFirst struct {
+	rr *roundRobin
+}
+
+// PickFirst Balancer is a simple balancer for testing multi-addresses in one addrConn.
+// By using this balancer, all address shares the same addrConn.
+// Although it wrapped by RoundRobin balancer, the logic of all methods work fine because
+// balancer.Get() returns the address Up by resetTransport()
+func PickFirst(r naming.Resolver) Balancer {
+	return &pickFirst{rr: &roundRobin{r: r}}
+}
+
+// The only difference is using ff.watchAddrUpdates() to use findFirstMD
+func (ff *pickFirst) Start(target string, config BalancerConfig) error {
```

Comment:
Oh, I'll change that. In my first implementation, I thought the word was findFirst, so I used ff. 
And also I forgot to update the comment because I was using metadata type(findFirstMD) to decide which balancer in that version.